streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/controller/RoleController.java

```diff
@@ -72,12 +72,12 @@ public RestResponse checkRoleName(@NotBlank(message = "{required}") String roleN
   @Operation(summary = "List role menus")
   @PostMapping("menu")
   public RestResponse getRoleMenus(@NotBlank(message = "{required}") String roleId) {
-    List<RoleMenu> list = this.roleMenuServie.listByRoleId(roleId);
-    List<String> roleMenus =
-        list.stream()
+    List<RoleMenu> roleMenuList = this.roleMenuServie.listByRoleId(roleId);
```

Comment:
roleIdMenuList？If you write the id in the back, this way is more appropriate, but it is easy to confuse before and after, I think you can consider changing the form